### PR TITLE
Added weather and discourse posts commands

### PR DESCRIPTION
**Pull request by vidhanm**

### CODE REVIEW

File: bot/events.py
Comment (kashifulhaque) [inline bot/events.py:1]: Don't remove that `async with message.channel.typing():` line of code. It is required to show the "typing..." indicator on Discord

File: commands/general.py
Comment (kashifulhaque) [inline commands/general.py:1]: Make a separate service where you call the weather API. Don't hardcode the API key.

**Review by kashifulhaque — CHANGES_REQUESTED** [review]
Please make the required changes.

File: bot/events.py
Comment (ReDxDaGer) [reply bot/events.py:1]: ohh okayy got it !!

**Pull request merged**